modules/installation-configuration-parameters.adoc

```diff
@@ -873,7 +939,7 @@ For a private cluster, specify a private subnet for each availability zone.
 For clusters that use AWS Local Zones, you must add AWS Local Zone subnets to this list to ensure edge machine pool creation.
 |Valid subnet IDs.
 
-|`PreserveBootstrapIgnition`
+|PreserveBootstrapIgnition:
```

Comment:
To maintain consistency: `platform: aws: PreserveBootstrapIgnition:` as per https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml ?

Reply:
you're correct on that, thank you